spec/db.spec.js

```diff
@@ -144,6 +151,41 @@ describe('db', () => {
           });
         });
 
+        describe('.getTableCreateScript', () => {
+          it('should return table create script', async() => {
+            const [createScript] = await dbConn.getTableCreateScript('users');
+
+            if (dbClient === 'mysql') {
+              expect(createScript).to.contain('CREATE TABLE `users` (\n' +
```

Comment:
You are right. If it depends on the output format then template string is a bad choice. Is gonna look to weird with all content moved to left.